src/jdk.httpserver/share/classes/com/sun/net/httpserver/HttpExchange.java

```diff
@@ -36,30 +36,36 @@
  * response to be generated in one exchange. It provides methods
  * for examining the request from the client, and for building and
  * sending the response.
- * <p>
- * The typical life-cycle of a HttpExchange is shown in the sequence
- * below.
- * <ol><li>{@link #getRequestMethod()} to determine the command
- * <li>{@link #getRequestHeaders()} to examine the request headers (if needed)
- * <li>{@link #getRequestBody()} returns a {@link java.io.InputStream} for reading the request body.
- *     After reading the request body, the stream should be closed.
- * <li>{@link #getResponseHeaders()} to set any response headers, except content-length
- * <li>{@link #sendResponseHeaders(int,long)} to send the response headers. Must be called before
- * next step.
- * <li>{@link #getResponseBody()} to get a {@link java.io.OutputStream} to send the response body.
- *      When the response body has been written, the stream must be closed to terminate the exchange.
+ *
+ * <p> The typical life-cycle of a {@code HttpExchange} is shown in the sequence
+ * below:
+ * <ol>
+ *     <li>{@link #getRequestMethod()} to determine the command.
+ *     <li>{@link #getRequestHeaders()} to examine the request headers (if
+ *     needed).
+ *     <li>{@link #getRequestBody()} returns a {@link java.io.InputStream} for
```

Comment:
Hi Patrick, could you double check what the text of the link is in the generated javadoc here? Is it a `java.io.InputStream` or just a `InputStream` (in which case `a` should be `an`)

Reply:
It appears as `InputStream` so I've changed `a` to `an` as suggested in new commit (3b4546e).